model/config.go

```diff
@@ -2779,6 +2784,12 @@ func (ls *LdapSettings) isValid() *AppError {
 				return NewAppError("ValidateFilter", "ent.ldap.validate_filter.app_error", nil, err.Error(), http.StatusBadRequest)
 			}
 		}
+
+		if *ls.GuestFilter != "" {
+			if _, err := ldap.CompileFilter(*ls.GuestFilter); err != nil {
+				return NewAppError("ValidateFilter", "ent.ldap.validate_guest_filter.app_error", nil, err.Error(), http.StatusBadRequest)
```

Comment:
Typically the "where" parameter of `NewAppError` uses the name of the surrounding function/method like so:

```suggestion
				return NewAppError("isValid", "ent.ldap.validate_guest_filter.app_error", nil, err.Error(), http.StatusBadRequest)
```
or
```suggestion
				return NewAppError("LdapSettings.isValid", "ent.ldap.validate_guest_filter.app_error", nil, err.Error(), http.StatusBadRequest)
```

Reply:
Ah.  Thanks for this.